Solutions/Auth0/Data Connectors/Auth0_FunctionApp.json

```diff
@@ -0,0 +1,135 @@
+		},
+		{
+			"title": "",
+			"description": "**2. Configure the Function App**\n\n1. In the Function App, select the Function App Name and select **Configuration**.\n2. In the **Application settings** tab, select ** New application setting**.\n3. Add each of the following application settings individually, with their respective string values (case-sensitive): \n\t\tDOMAIN\n\t\tCLIENT_ID\n\t\tCLIENT_SECRET\n\t\tWorkspaceID\n\t\tWorkspaceKey\n\t\tlogAnalyticsUri (optional)\n> - Use logAnalyticsUri to override the log analytics API endpoint for dedicated cloud. For example, for public cloud, leave the value empty; for Azure GovUS cloud environment, specify the value in the following format: `https://<CustomerId>.ods.opinsights.azure.us`.\n4. Once all application settings have been entered, click **Save**."
+		}
+	],
+	"metadata": {
+        "id": "056770e6-8e29-4341-8239-4b38fcd9f53d",
+        "version": "1.0.0",
+        "kind": "dataConnector",
+        "source": {
+            "kind": "community"
+        },
+        "author": {
+            "name": "Microsoft"
+        },
+        "support": {
+            "tier": "community",
+            "name": "Microsoft",
+            "link":"https://github.com/Azure/Azure-Sentinel/issues"
```

Comment:
@ospeka : Could you please change link ? current link points to github issues it seems to be incorrect